src/handlers/wildcard/unassign.ts

```diff
@@ -72,7 +72,8 @@ const checkBountyToUnassign = async (issue: Issue): Promise<boolean> => {
         );
       } else
         await addCommentToIssue(
-          `${askUpdate} @${assignees[0]}? If you would like to release the bounty back to the DevPool, please comment \`/unassign\` \nLast activity time: ${lastActivity}`,
+          `${askUpdate} @${assignees[0]}? If you would like to release the bounty back to the DevPool, please comment \`
```

Comment:
I think there is a mistake here @me505 @0xcodercrane

Reply:
good catch!

Reply:
fixed. ➡️ https://github.com/ubiquity/ubiquibot/commit/b742637e3f95b91e0df0b36f0735d05f5b746dac